sdk/storage/azure-storage-blob/azure/storage/blob/blob_service_client.py

```diff
@@ -0,0 +1,469 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import functools
+from typing import (  # pylint: disable=unused-import
+    Union, Optional, Any, Iterable, Dict, List,
+    TYPE_CHECKING
+)
+try:
+    from urllib.parse import urlparse
+except ImportError:
+    from urlparse import urlparse
+
+from ._shared.shared_access_signature import SharedAccessSignature
+from ._shared.models import LocationMode
+from ._shared.utils import (
+    StorageAccountHostsMixin,
+    return_response_headers,
+    parse_connection_str,
+    process_storage_error,
+    parse_query)
+from ._generated import AzureBlobStorage
+from ._generated.models import StorageErrorException, StorageServiceProperties
+from .container_client import ContainerClient
+from .blob_client import BlobClient
+from .models import ContainerProperties, ContainerPropertiesPaged
+
+if TYPE_CHECKING:
+    from datetime import datetime
+    from azure.core import Configuration
+    from azure.core.pipeline.transport import HttpTransport
+    from azure.core.pipeline.policies import HTTPPolicy
+    from ._shared.models import AccountPermissions, ResourceTypes
+    from .lease import LeaseClient
+    from .models import (
+        BlobProperties,
+        Logging,
+        Metrics,
+        RetentionPolicy,
+        StaticWebsite,
+        CorsRule
+    )
+
+
+class BlobServiceClient(StorageAccountHostsMixin):
+    """ A client interact with the Blob Service at the account level.
+
+    This client provides operations to retrieve and configure the account properties
+    as well as list, create and delete containers within the account.
```

Comment:
Minor: I noticed that Oxford comma is not being used, I just want to confirm that this is indeed intentional and consistent across the docs.